predicate/predicate_bytes.go

```diff
@@ -62,3 +62,7 @@ func GetPredicateResultBytes(extraData []byte) ([]byte, bool) {
 	}
 	return extraData[params.DynamicFeeExtraDataSize:], true
 }
+
+func SetPredicateResultBytes(extraData []byte, predicateResults []byte) []byte {
+	return append(extraData[:params.DynamicFeeExtraDataSize], predicateResults...)
```

Comment:
What is extra data supposed to be that we know the prefix? Please add a comment describing the parameter.

Reply:
The comments help; thanks.